surefire-providers/common-java5/src/main/java/org/apache/maven/surefire/report/SmartStackTraceParser.java

```diff
@@ -289,31 +276,48 @@ static boolean containsClassName( StackTraceElement[] stackTrace, String classNa
         return false;
     }
 
-    public static String causeToString( Throwable cause )
+    private static String causeToString( Throwable cause, StackTraceFilter filter )
     {
-        StringBuilder resp = new StringBuilder();
+        String resp = "";
         while ( cause != null )
         {
-            resp.append( "Caused by: " );
-            resp.append( toString( cause, Arrays.asList( cause.getStackTrace() ) ) );
+            resp += "Caused by: ";
```

Comment:
no, no. We are on Java 5 and therefore the bytecode will contain StringBuilder again. IntelliJ IDEA also notifies to remove StringBuilder in the code for simplicity reasons. The result will be the same on runtime.